lib/scsbxml_fetcher.rb

```diff
@@ -1,8 +1,10 @@
 require 'oauth2'
 require 'httparty'
 require 'nypl_log_formatter'
+require './lib/errorable'
```

Comment:
Can you change this to `File.join('.', 'lib', 'errorable')` ?
It will use `File::SEPARATOR`, which, depending on the OS may not be `/`.

Reply:
Sure! I missed this.